### ISSUE TITLE
Enhancement: Use a controller to render the form, success and failure pages

### ISSUE THREAD

**Issue by robbieaverill** [Firesphere/silverstripe-stripeslack#8]
To implement the module at the moment you need to add a new page type which has a template that renders the form. You'd then also need to add a new page for success and error pages after submitting the form.

Since you have to add template code for this anyway, it would be nice if a controller with three separate actions (and templates) could be used instead so I just need to modify the templates to create the form and the success/error pages.

**Comment by Firesphere** [comment]
It _is_ already using the Controller, but just via an extension. On the latest feature/updates branch, I've added the shortcode (although that's an iffy one, as the form is wrapped in a <p> tag, which isn't allowed)

I guess an example page type and defaults for that would be a fair backup? I don't want to enforce new PageTypes, as the reason I went initially with using an Extension, was I expected it to be used in a Sidebar or Footer.
